manager/templates/default/header.tpl

```diff
@@ -23,13 +23,56 @@
 <script src="{$_config.connectors_url}lang.js.php?ctx=mgr&topic=topmenu,file,resource,{$_lang_topics}&action={$smarty.get.a|default|htmlspecialchars}"></script>
 <script src="{$_config.connectors_url}modx.config.js.php?action={$smarty.get.a|default|htmlspecialchars}{if $_ctx}&wctx={$_ctx}{/if}&HTTP_MODAUTH={$_authToken|default|htmlspecialchars}"></script>
 
+<script>
+    const tvPanelOverrides = [];
+</script>
+
 {$maincssjs}
 {foreach from=$cssjs item=scr}
 {$scr}
 {/foreach}
 
 <script>
     MODx.config.search_enabled = {$_search};
+    if (!Ext.isEmpty(tvPanelOverrides)) {
```

Comment:
Would it be possible to return "early" instead if this huge `if` block?

I'm not sure if this is possible in this context.
```
if (Ext.isEmpty(tvPanelOverrides)) {
     return;
}
```

Reply:
I don't see how that would make a difference, as the if block is not going to be evaluated when tvPanelOverrides is empty. Also, it's not a function, so there's nothing to return.